### PR TITLE
[BEAM-2736] Suppress Attempted Metrics tests in Dataflow Batch

### PR DESCRIPTION
**Pull request by tgroh**
Batch does not support attempted metrics, only committed and tentative.

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by tgroh** [comment]
R: @kennknowles

**Comment by tgroh** [comment]
run dataflow validatesrunner

File: runners/google-cloud-dataflow-java/pom.xml [hunk 1]
```diff
@@ -245,6 +245,7 @@
               -->
               <excludedGroups>
                 org.apache.beam.sdk.testing.LargeKeys$Above10MB,
+                org.apache.beam.sdk.testing.UsesAttemptedMetrics,
```
Comment (bjchambers) [inline runners/google-cloud-dataflow-java/pom.xml]: This doesn't seem to be exclusively Dataflow Batch. How does this affect Streaming tests?

Comment (tgroh) [inline runners/google-cloud-dataflow-java/pom.xml]: We don't execute streaming tests with this configuration, and when we do we'll have a second configuration. See https://github.com/apache/beam/pull/2947

**Comment by tgroh** [comment]
The validates runner failure is expected; importantly is the fact that the tests which are not applicable are now succeeding.

+R: @bjchambers

**Comment by kennknowles** [comment]
LGTM

**Pull request closed without merging**